src/Troubleshooting/index.md

```diff
@@ -8,6 +8,79 @@ This is due to a Roblox update causing an incompatibility with Wayland's handlin
 
 #### White Screen
 
-This specific problem can be the same as many other root causes, a common one is your graphics card not supporting Vulkan, or is missing the graphics libraries required for Wine; you can see [Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md)
+This specific problem can be the same as many other root causes, a common one is your system missing the graphics libraries required for Wine; you can check [Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md) out for more information.
 
-This may also be your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
+This may also indicate your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
+
+#### Make sure there are no stale wineservers running
+
+There are multiple reasons why this'd occur, but `vinegar kill` will suffice in order to fix the issue.
+
+#### Roblox Studio stuck on the splash
+
+This usually happens due to issues with the DPI. In order to fix it, set your DPI to 97 by running `vinegar exec winecfg` to open the wine configuration, and then heading to the Graphics tab. You should be able to find a DPI option at the bottom of the page.
+
+
+The only downside of this is that the Roblox fonts will be a bit blurry. You could try setting it to an even number larger than 96, but this'd cause in a bigger interface within wine.
+
+#### Unable to log in through the Studio or Player apps
+
+This occurs due to Webview2 issues with wine.
+
+There are 2 workarounds:
+
+- Quick Login (Player): Click on "Log in with another device", and simply enter the code that displays on your screen through another device (or the website).
+  
+- Website (Player and Studio): Log in through the [Roblox website](https://roblox.com) and launch/edit an experience.
+
+#### "Your GPU is incompatible" error when launching Roblox Studio
+
+This issue is most commonly caused by your wine build not having the Vulkan ChildWindow patch applied. To fix it, you could either:
+
+- Use a Wine build that has the ChildWindow patch enabled (such as [Wine GE](https://github.com/GloriousEggroll/wine-ge-custom)).
+
+- Disable DXVK in the Vinegar configuration.
+
+#### Memory Dump
+
+This is because of a  [commit](https://gitlab.winehq.org/wine/wine/-/commit/ea640f6cece7660ffc853b7d574fbe52af34901a) (on Wine 8.16 and newer) causing Roblox to create a memory dump.
```

Comment:
```suggestion
This is because of [commit ea640f6c](https://gitlab.winehq.org/wine/wine/-/commit/ea640f6cece7660ffc853b7d574fbe52af34901a) (on Wine 8.16 and newer) causing Roblox to create a memory dump.
```